misc/scratch/big-dev-box.json

```diff
@@ -2,7 +2,7 @@
     "name": "one-off load testing",
     "launch_script": "true",
     "instance_type": "r5a.4xlarge",
-    "ami": "ami-0aeb7c931a5a61206",
+    "ami": "ami-09d56f8956ab235b3",
```

Comment:
Can you update ci.json and persist-benchmarking.json the same way? You can just delete release.json; it's no longer in use. Even better would be to change this to take a dict like:
```json
{
  "ami": {
    "us-east-1": "ami-...",
    "us-east-2": "ami-...",
  }
}
```
So that we can support running the configuration in whatever regions are listed in the map.

Reply:
Agree about the dict, but I'm going to leave that to someone else.